Documentation/LanguageServer/Windows Subsystem for Linux.md

```diff
@@ -1,75 +1,131 @@
-For developers using the Windows Subsystem for Linux, we recommend you start with the following **c_cpp_properties.json** template.  Select "C/Cpp: Edit Configurations" from the command palette to create this file if you haven't already.
+# Windows Subsystem for Linux
 
-```
-        {
-            "name": "WSL",
-            "intelliSenseMode": "clang-x64",
-            "includePath": [
-                "${workspaceRoot}",
-                "${localappdata}/lxss/rootfs/usr/include/c++/5",
-                "${localappdata}/lxss/rootfs/usr/include/x86_64-linux-gnu/c++/5",
-                "${localappdata}/lxss/rootfs/usr/include/c++/5/backward",
-                "${localappdata}/lxss/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include",
-                "${localappdata}/lxss/rootfs/usr/local/include",
-                "${localappdata}/lxss/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include-fixed",
-                "${localappdata}/lxss/rootfs/usr/include/x86_64-linux-gnu",
-                "${localappdata}/lxss/rootfs/usr/include"
-            ],
-            "defines": [
-                "__linux__",
-                "__x86_64__"
-            ],
-            "browse": {
-                "path": [
-                    "${localappdata}/lxss/rootfs/usr/include/c++/5",
-                    "${localappdata}/lxss/rootfs/usr/include/x86_64-linux-gnu/c++/5",
-                    "${localappdata}/lxss/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include",
-                    "${localappdata}/lxss/rootfs/usr/local/include",
-                    "${localappdata}/lxss/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include-fixed",
-                    "${localappdata}/lxss/rootfs/usr/include/x86_64-linux-gnu",
-                    "${localappdata}/lxss/rootfs/usr/include/*"
-                ],
-                "limitSymbolsToIncludedHeaders": true,
-                "databaseFilename": ""
-            }
-        }
+To use the Windows Subsystem for Linux with this extension you need to add a configuration to your **c_cpp_properties.json** file which adds the necessary header paths from within the WSL filesystem to the `includePath`.
+
+Select "C/Cpp: Edit Configurations" from the command palette to create the **c_cpp_properties.json** file if you haven't already.
+
+## Release
+
+For developers using Ubuntu with the current version of WSL released with the Fall Creators Update, you can add the following configuration template to your **c_cpp_properties.json** file.  
 
+```json
+{
+    "name": "WSL",
+    "intelliSenseMode": "clang-x64",
+    "includePath": [
+        "${workspaceRoot}",
+        "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/c++/5",
+        "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/x86_64-linux-gnu/c++/5",
```

Comment:
So this works for Ubuntu, but what about OpenSUSE and SLES which are also available in the Windows Store?  Should we provide configs for each of them?